simple/src/myPakage/FindDestinationCity.java

```diff
@@ -0,0 +1,45 @@
+package myPakage;
+
+import java.util.ArrayList;
+import java.util.LinkedList;
+import java.util.List;
+
+public class FindDestinationCity {
+    public static void main(String[] args) {
+        List pairOne = new LinkedList<>();
+        pairOne.add("London");
+        pairOne.add("New York");
+        List PairTwo = new LinkedList<>();
+        PairTwo.add("New York");
+        PairTwo.add("Lima");
+        List pairThree = new LinkedList<>();
+        pairThree.add("Lima");
+        pairThree.add("Sao Paulo");
+        List<List<String>> commonPair = new ArrayList<>();
```

Comment:
try adding cities with different approach, no need to create separate list object for each route

Reply:
done